### PR TITLE
resolvectl: add support for various new flags

### PR DESCRIPTION
**Pull request by poettering**
split out of #17535

### CODE REVIEW

File: man/resolvectl.xml [hunk 1]
```diff
@@ -262,6 +262,65 @@
         returned.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--validate=</option><replaceable>BOOL</replaceable></term>
+
+        <listitem><para>Takes a boolean parameter; used in conjunction with <command>query</command>. If true
+        (the default), DNSSEC validation is applied as usual — under the condition it is enabled for the
+        network and for <filename>systemd-resolved.service</filename> as a whole. If false, DNSSEC validation
+        is disabled for the specific query, regardless if enabled for the network or in the service. Note
```
Comment (bluca) [inline man/resolvectl.xml]: "regardless of whether it is enabled"

File: man/resolvectl.xml [hunk 2]
```diff
@@ -262,6 +262,65 @@
         returned.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--validate=</option><replaceable>BOOL</replaceable></term>
+
+        <listitem><para>Takes a boolean parameter; used in conjunction with <command>query</command>. If true
+        (the default), DNSSEC validation is applied as usual — under the condition it is enabled for the
```
Comment (bluca) [inline man/resolvectl.xml]: "under the condition that"

**Comment by poettering** [comment]
Made the suggested changes and rebased. No other changes.

**Comment by yuwata** [comment]
LGTM.

**Pull request merged**